conn2res/performance.py

```diff
@@ -0,0 +1,381 @@
+        
+        ‘raw_values’ :
+        Returns a full set of scores in case of 
+        multioutput input.
+
+        ‘uniform_average’ :
+        Scores of all outputs are averaged with 
+        uniform weight.
+
+    Returns
+    -------
+    error : float or ndarray of floats.
+        A floating point value or an array of floating 
+        point values, one for each individual target.
+    """
+    func = getattr(metrics, 'mean_squared_error')
+    return func(y_true, y_pred, sample_weight=sample_weight, multioutput=multioutput, squared=True)
+
+
+def root_mean_squared_error(y_true, y_pred, sample_weight=None, multioutput='uniform_average', normalize=False, **kwargs):
+    """
+    Root mean squared error. If normalize is True, the error is
+    normalized by the variance of y_true.
+
+    Parameters
+    ----------
+    y_true : numpy.ndarray
+        Ground truth target values.
+    y_pred : numpy.ndarray
+        Predicted target values.
+    sample_weight : numpy.ndarray 
+        Sample weights.
+    multioutput : str 
+        Defines aggregating of multiple output scores:
+        
+        ‘raw_values’ :
+        Returns a full set of scores in case of 
+        multioutput input.
+
+        ‘uniform_average’ :
+        Scores of all outputs are averaged with 
+        uniform weight.
+    normalize : bool
+        If True normalizes error by variance of y_true
+
+    Returns
+    -------
+    error : float or ndarray of floats.
+        A floating point value or an array of floating 
+        point values, one for each individual target.
+    """
+    func = getattr(metrics, 'mean_squared_error')
+    if normalize: 
+        return func(y_true, y_pred, sample_weight=sample_weight, multioutput=multioutput, squared=False) / y_true.var(axis=0)
+    else: 
+        return func(y_true, y_pred, sample_weight=sample_weight, multioutput=multioutput, squared=False)
+
+
+def mean_absolute_error(y_true, y_pred, sample_weight=None, multioutput='uniform_average', **kwargs):
+    """
+    Mean absolute error.
+
+    Parameters
+    ----------
+    y_true : numpy.ndarray
+        Ground truth target values.
+    y_pred : numpy.ndarray
+        Predicted target values.
+    sample_weight : numpy.ndarray 
+        Sample weights.
+    multioutput : str 
+        Defines aggregating of multiple output scores:
+        
+        ‘raw_values’ :
+        Returns a full set of scores in case of 
+        multioutput input.
+
+        ‘uniform_average’ :
+        Scores of all outputs are averaged with 
+        uniform weight.
+
+    Returns
+    -------
+    error : float or ndarray of floats.
+        A floating point value or an array of floating 
+        point values, one for each individual target.
+    """
+    func = getattr(metrics, 'mean_absolute_error')
+    return func(y_true, y_pred, sample_weight=sample_weight, multioutput=multioutput)
+
+
+def corrcoef(y_true, y_pred, multioutput='uniform_average', nonnegative=None, **kwargs): 
```

Comment:
As the default value is nonnegative=None, you will need to update e.g., workflows.py so that it uses metric_kws={'nonnegative': 'squared'} to be able to produce the same results as before.